src/Fixer/Basic/BracesFixer.php

```diff
@@ -169,9 +169,17 @@ protected function createConfigurationDefinition()
             ->getOption()
         ;
 
+        $positionAfterControlStructures = new FixerOptionBuilder('position_after_control_structures', 'whether the opening brace should be placed on "next" or "same" line after control structures not covered by position_after_functions_and_oop_constructs.');
```

Comment:
I would remove the _"not covered by position_after_functions_and_oop_constructs"_ part.